.env

```diff
@@ -98,4 +98,15 @@ _APP_VCS_GITHUB_CLIENT_SECRET=
 _APP_VCS_GITHUB_WEBHOOK_SECRET=
 _APP_MIGRATIONS_FIREBASE_CLIENT_ID=
 _APP_MIGRATIONS_FIREBASE_CLIENT_SECRET=
-_APP_ASSISTANT_OPENAI_API_KEY=
\ No newline at end of file
+_APP_ASSISTANT_OPENAI_API_KEY=
+_APP_MESSAGE_SMS_PROVIDER_MSG91_SENDER_ID=
+_APP_MESSAGE_SMS_PROVIDER_MSG91_AUTH_KEY=
```

Comment:
Since the send message endpoints are generic for each provider type (SMS, email and APNS), only one provider type's env variables are needed for test cases.

Reply:
I like idea of having some kind of `TESTS` prefix. Still, it would be cool to use DSN I think, allowing us to have it all in 1 variable.